src/scripts/views/form/form-map.js

```diff
@@ -0,0 +1,109 @@
+import page            from 'page';
+import Action          from '../../actions';
+import UserAction      from '../../actions/user';
+import BroadcastAction from '../../actions/broadcast';
+
+/**
+ *
+ */
+
+export default
+	{
```

Comment:
Rather have the leading bracket stay on the same row as the `export default` expression. Like so:
```
export default {
  // ...
}
```